src/services/user.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '../types/User';
+import usersFromServer from '../api/users';
+
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
```

Comment:
The function returns `null` if no user is found, which is a good practice to handle missing data. Ensure that any component using this function checks for `null` to avoid rendering issues.